mattermost-plugin/webapp/src/plugin.scss

```diff
@@ -9,9 +9,6 @@
     z-index: 1000;
 
     .Menu {
-        position: unset;
```

Comment:
@sbishel, was there a reason this was needed in the original checkin? I'm worried removing it might break something else?

Reply:
The `min-width: unset;` is not needed because the `min-width: 180px;` in `menu.scss` is being removed as well, see below.

However, the `position: unset;` removal does cause a slight difference in menu positioning between `Channels` and `Boards`.

<img width="933" alt="Screen Shot 2021-09-13 at 5 23 11 PM" src="https://user-images.githubusercontent.com/12704875/133169835-3cef2cd4-be66-4e9a-8179-f2e27646fed2.png">
<img width="947" alt="Screen Shot 2021-09-13 at 5 23 01 PM" src="https://user-images.githubusercontent.com/12704875/133169838-2a638df8-c352-4e4c-aa84-673f15b3cb8a.png">